roles/reverseproxy/tasks/main.yaml

```diff
@@ -1,44 +1,45 @@
-- name: Install caddy
```

Comment:
Should we add a task which deactivates and uninstally caddy first? Otherwise we end up with two reverse proxies (which will obviously give a conflict).

Reply:
Cleanup isn't the most desirable thing to do in ansible, would delete it by hand, otherwise i would have a commit to add the delete role, run it, and then have to have a commit to delete the role (and thats basically one line with a purge in it).

If you want it i can do that too, but i would rather not run cleanup tasks in ansible due to velocity.

Reply:
I'm fine with that.